nixos/modules/services/web-apps/akkoma.nix

```
@@ -1136,6 +1180,6 @@ in {
     };
   };
 
-  meta.maintainers = with maintainers; [ mvs ];
+  meta.maintainers = with lib.maintainers; [ mvs tcmal ];
```

```suggestion
  meta.maintainers = with lib.maintainers; [ mvs ];
```
tcmal is not in the maintainers list since #320858